app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1423,6 +1423,11 @@ public void onPlayerError(@NonNull final PlaybackException error) {
             case ERROR_CODE_TIMEOUT:
             case ERROR_CODE_IO_UNSPECIFIED:
             case ERROR_CODE_IO_NETWORK_CONNECTION_FAILED:
+                isCatchableException = true;
+                simpleExoPlayer.pause();
+                onPaused();
+                ErrorUtil.showErrorToast(context, "No internet connection available");
```

Comment:
You must create a string ressource and reference it here, to allow translation of this message.